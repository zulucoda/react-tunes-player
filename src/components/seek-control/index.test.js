import React from 'react';
import { render, screen } from '@testing-library/react';
import SeekControl from '.';
import '@testing-library/jest-dom/extend-expect';

describe('Components - Seek Controls - Render Test', () => {
  test('should render seek controls', () => {
    const { container } = render(<SeekControl />);
    expect(container).toMatchSnapshot();
  });
});
