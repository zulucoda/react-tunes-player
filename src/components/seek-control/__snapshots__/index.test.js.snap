// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Components - Seek Controls - Render Test should render seek controls 1`] = `
<div>
  <div
    class="sc-bczRLJ jzXAAq seek-control"
    role="seek-control"
  >
    <span
      class="sc-gsnTZi ktPVQr seek-control-time-progress"
      role="seek-control-time-progress"
    >
      0
    </span>
    <progress
      class="sc-dkzDqf fasqwe seek-control-progress"
    />
    <span
      class="sc-gsnTZi ktPVQr seek-control-time-total"
      role="seek-control-time-total"
    >
      0
    </span>
  </div>
</div>
`;
